package.json

```diff
@@ -6,7 +6,9 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "prepare": "husky install",
```

Comment:
The `prepare` script is run anytime we call `yarn install` (or just `yarn`), so this guarantees that `husky` hooks will always be installed on each contributor's machine.